### DISCUSSION TITLE
How to reflash the SD card as if it was the 1st time?

### DISCUSSION THREAD

**Discussion by luisgois** [tesla-android/issue-tracker#16] [General]
Something I've noticed is that the 1st time I use an SD card with the "fastboot flash" commands, I get almost no errors, e.g.:

>fastboot flash bootloader bootloader-sd.img
Invalid sparse file format at header magic
Sending sparse 'bootloader' 1/1 (9056 KB)          OKAY [  0.253s]
Writing 'bootloader'                               OKAY [  7.634s]
Finished. Total time: 8.007s

But if I re-flash it a second time (even using diskpart on Windows to wipe clean the partition table before using balenaEtcher), the the fasboot flash commands start failing and they take much longer to run, e.g.

>fastboot flash bootloader bootloader-sd.img
Invalid sparse file format at header magic
Sending sparse 'bootloader' 1/1 (9056 KB)          FAILED (Error reading sparse file)
fastboot: error: Command failed

Thanks in advance

**Comment by luisgois** [comment]
Did a few more tests, using new SD cards, on multiple OSes. Windows is the most consistent. USB port used matters a lot, but needs to provide enough juice to power the RPi.

**Reply by mikegapinski** [reply]
next update will include a better power management implementation. This is the biggest flop of the first alpha, the cpu should not be fixed on the highest clock at all times
